pkg/core/converters.go

```diff
@@ -427,3 +427,12 @@ func ExtractTransactions(id tongo.BlockIDExt, block *tlb.Block) ([]*Transaction,
 	}
 	return transactions, nil
 }
+
+func ConvertJettonMeta(metadata tongo.JettonMetadata) JettonMetadata {
+	return JettonMetadata{
```

Comment:
maybe we can use tongo.MetaData directly?

Reply:
why do we need local JettonMetadata type?

Reply:
✅